akka-slf4j/src/test/scala/akka/event/slf4j/Slf4jLoggerSpec.scala

```diff
@@ -155,6 +155,14 @@ class Slf4jLoggerSpec extends AkkaSpec(Slf4jLoggerSpec.config) with BeforeAndAft
       s should include("msg=[Message with custom MDC values]")
     }
 
+    "support null marker" in {
+      producer ! StringWithMarker("security-wise interesting message", null)
```

Comment:
It would be cool to reproduce it via `log.warn(cause, ...)` as well. Optional IMHO though.

Reply:
Well, the test already exists but somehow does not fail ....

https://github.com/akka/akka/blob/9941f7d80086cb602fdaac86feabbe82733731aa/akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/ActorLoggingSpec.scala#L124

Reply:
Aah, that test tests via the `akka.testkit.TestEventListener`, so it wouldn't see any problems in `Slf4jLogger`. OK then :+1: